vercel.json

```diff
@@ -1,10 +1,7 @@
 {
-  "buildCommand": "cd packages/core/helper-plugin && yarn build-storybook",
   "github": {
     "silent": true,
     "autoJobCancelation": true
   },
-  "installCommand": "yarn install --immutable",
   "ignoreCommand": "git diff HEAD^ HEAD --quiet './packages/core/helper-plugin'",
```

Comment:
I think you can get rid of this because this only really applied to the `helper-plugin` storybook

Reply:
The ignoreCommand?